pkg/features/README.md

```diff
@@ -0,0 +1,50 @@
+# Feature Flags
+
+You can use feature flags to control visibility of a feature or a block of code. The end user will not see the feature unless the flag is enabled.
+With this, you can control the release of said feature or allow a user to enable/disable some functionality in production.
+Currently, feature flags use environment variables to power the toggles.
+
+Feature flags can be valuable to ship features in a preview state, to provide the end user a way to disable some functionality or to control any boolean setting.
+
+> :warning: Feature flags cannot be used inside migrations or schema changes. In other words any such change will always be applied regardless of any feature flag value.
+
+## Adding a feature flag
+
+To add a feature flag, add a variable with your feature to `list.go`. The variable can be one of two types of feature flag:
+
+#### An unchangeable feature:
+This flag cannot be changed from its default value on release builds (i.e. "production"). To enable or disable it, you must make a code change.
+On development builds, the setting _can_ be changed.
```

Comment:
For those in the context it's well known that one needs to set an environment variable value but those without the context will have to spend time looking for the answer.

For anyone who comes here without any prior context, could you please clarify what real action needs to be taken by an external user or a developer to _change_ the setting?

Reply:
(going off-topic) I imagine it's possible to store Central runtime-mutable feature flags in DB. We just need to make sure the precedence of values between explicitly set environment variables and values from DB is not causing surprises. That is to say: if I, as a user, set environment variable but the runtime value is different because it came from runtime database previously set by someone, I'd likely be surprised. Therefore, I'll suggest the order of resolving a mutable value should be: (1) env variable, if present, (2) db value, if present, (3) default. The UI part responsible for setting DB values should let users know when env variable is present that it's value has precedence.